yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -381,6 +382,19 @@ func deletePGReplicationSlot(msr *metadb.MigrationStatusRecord, source *srcdb.So
 	}
 }
 
+func deletePGPublication(msr *metadb.MigrationStatusRecord, source *srcdb.Source) {
+	if msr.PGPublicationName == "" || source.DBType != POSTGRESQL {
+		log.Infof("pg publication name is not set or source db type is not postgresql. skipping deleting pg publication name")
```

Comment:
nit: it'll be also valuable to have the log for the inverse case: i.e. when we are indeed deleting the publication name.